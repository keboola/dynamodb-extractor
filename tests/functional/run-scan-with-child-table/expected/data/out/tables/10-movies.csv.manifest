{
  "incremental": true,
  "columns": [
    "title",
    "info"
  ],
  "column_metadata": {
    "title": [
      {
        "key": "KBC.datatype.nullable",
        "value": true
      },
      {
        "key": "KBC.datatype.basetype",
        "value": "string"
      }
    ],
    "info": [
      {
        "key": "KBC.datatype.nullable",
        "value": true
      },
      {
        "key": "KBC.datatype.basetype",
        "value": "string"
      }
    ]
  }
}