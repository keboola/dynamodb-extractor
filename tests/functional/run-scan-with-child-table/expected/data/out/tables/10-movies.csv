"Transformers: Age of Extinction","d008089a1cb90697b3bbef6941c29624"
"X-Men: Days of Future Past","f1f4064e7780cc73cefd9bf3e3ec5e5c"
"Insidious: Chapter 2","0573c5cea38a536e2ea911e1d23ac083"
"Now You See Me","4245e6dcfce5a9275511ec172690977e"
"Prisoners","9b20dbf9a7f06306ef49368827ffa11b"
"Rush","a214d738e894ec7c6b4137f5265daa9a"
"The Hunger Games: Catching Fire","ef32318cfef9aa96c9ceb973a7b3a642"
"This Is the End","94ecb1a4a5b002e7063f78d9b60cd041"
"Thor: The Dark World","7f7968accdac206fed62393066f51388"
"World War Z","084592b8967def33d25ef52abcc0b583"
